odpf/guardian/guardian.proto

```diff
@@ -0,0 +1,375 @@
+syntax = "proto3";
+package odpf.guardian;
+
+import "odpf/third_party/googleapis/google/api/annotations.proto";
+import "odpf/third_party/grpc-gateway/protoc-gen-openapiv2/options/annotations.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/odpf/proton/guardian";
+option java_multiple_files = true;
+option java_package = "io.odpf.proton.guardian";
+option java_outer_classname = "ServiceManager";
+
+// These annotations are used when generating the OpenAPI file.
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+    info: {
+        version: "0.1";
+    };
+    external_docs: {
+        description: "Guardian server";
+    }
+    schemes: HTTP;
+};
+
+// WARNING: This is still in active development and can have breaking changes
+service Guardian {
```

Comment:
It's a general guideline to add `Service` suffix to services to distinguish them from proto messages.